javascript-practice/src/javascripts/models/category.js

```diff
@@ -0,0 +1,15 @@
+import APIService from "../services/service"
+
+export default class CategoryModel {
+	constructor(name) {
```

Comment:
No need to put the name into the the params of class here. You must input the param whenever you init this class.